specs/latest/2.0/index.html

```diff
@@ -697,13 +697,13 @@ <h2 class="no-toc">Table of contents</h2>
   /* Buffer objects */
   // WebGL1:
   void bufferData(GLenum target, GLsizeiptr size, GLenum usage);
-  void bufferData(GLenum target, BufferDataSource? srcData, GLenum usage);
-  void bufferSubData(GLenum target, GLintptr dstByteOffset, BufferDataSource? srcData);
+  void bufferData(GLenum target, ArrayBuffer srcData, GLenum usage);
+  void bufferSubData(GLenum target, GLintptr dstByteOffset, ArrayBuffer srcData);
   // WebGL2:
-  void bufferData(GLenum target, ArrayBufferView srcData, GLenum usage, GLuint srcOffset,
-                  optional GLuint length = 0);
+  void bufferData(GLenum target, ArrayBufferView srcData, GLenum usage,
+                  optional GLuint srcOffset = 0, optional GLuint length = 0);
```

Comment:
Again since it sounds like the intent has changed, this overload (which matches an existing WebGL 1.0 entry point) should take "ArrayBufferView?".